src/components/pages/coach/ReportList/index.jsx

```diff
@@ -258,45 +284,48 @@ class ReportList extends React.Component{
 
     render() {
         // Get global state with login information
-        const { globalState, location } = this.props;
+        const { globalState, match } = this.props;
         //> Route protection
         // Only logged in uses can access this page
         if(!globalState.logged) return <Redirect to="/login"/>
         // If logged in but not coach
         if(globalState.logged && !globalState.coach) return <Redirect to="/dashboard"/> 
         
-        if(!location.state) return <Redirect to="/coach"/>
+        if(this.state.uid === false || this.state.exists === false) return <Redirect to="/coach"/>
 
         console.log(this.state);
```

Comment:
# please remove `console` (no-console)

In JavaScript that is designed to be executed in the browser, it's considered a best practice to avoid using methods on `console`. Such messages are considered to be for debugging purposes and therefore not suitable to ship to the client. In general, calls using `console` should be stripped before being pushed to production.

```js
console.log("Made it here.");
console.error("That shouldn't have happened.");
```

Examples of **incorrect** code for this rule:

```js
/* eslint no-console: "error" */

console.log("Log a debug level message.");
console.warn("Log a warn level message.");
console.error("Log an error level message.");
```

Reply:
Noted.
However, the console functions do not exist without reason. It is a really great, fully integrated development tool.